### PR TITLE
Fix Add to cart for variants

### PR DESCRIPTION
**Pull request by eduardogomez97**
Required by: https://github.com/doofinder/support/issues/2384

### CODE REVIEW

**Review by juan-gm — APPROVED** [review]

File: views/js/add-to-cart/doofinder-add_to_cart_ps16.js [hunk 1]
```diff
@@ -15,5 +15,13 @@
 //implementation of "add to cart" functionality for prestashop 1.6.x
 
 let doofinderManageCart = (cartOptions) => {
-    ajaxCart.add(cartOptions.productID, cartOptions.customizationID, undefined, undefined, cartOptions.quantity);
+
+    let id_product_attribute;
+    if (cartOptions.productID.includes('VAR-')) {
+        id_product_attribute = entry.id.replace('VAR-', '');
```
Comment (sofia-doofinder) [inline views/js/add-to-cart/doofinder-add_to_cart_ps16.js]: @eduardogomez97 solo una pregunta, ese `entry.id.replace` de donde sale el `entry`? en que momento se define? normalmente no sería `cartOptions.productID.replace('VAR-', '')`?

Comment (eduardogomez97) [reply views/js/add-to-cart/doofinder-add_to_cart_ps16.js]: Sí, se me ha quedado de pruebas que he hecho. Cambiado!

**Review by sofia-doofinder — APPROVED** [review]

**Review by sofia-doofinder — APPROVED** [review]

File: views/js/add-to-cart/doofinder-add_to_cart_ps16.js [hunk 2]
```diff
@@ -15,5 +15,16 @@
 //implementation of "add to cart" functionality for prestashop 1.6.x
 
 let doofinderManageCart = (cartOptions) => {
-    ajaxCart.add(cartOptions.productID, cartOptions.customizationID, undefined, undefined, cartOptions.quantity);
+
+    let id_product_cart;
```
Comment (davidmolinacano) [inline views/js/add-to-cart/doofinder-add_to_cart_ps16.js]: In JS, I would change it to camelCase instead of using snake_case :smile:

**Review by davidmolinacano — APPROVED** [review]
Apart from the comment I've just wrote, everything else LGTM :trophy: Very good job!! :clap:

**Pull request merged**